test/inc/TestDef.h

```diff
@@ -14,3 +15,34 @@ static const std::wstring c_testFileExtension_Packaged = L".reuniontestfile-pack
 static const std::wstring c_testFilePhaseEventName = L"ReunionTestFilePhaseEventName";
 
 static const std::wstring c_testStartupPhaseEventName = L"ReunionTestStartupPhaseEventName";
+
+static const std::wstring c_testInstanceRedirectedPhaseEventName = L"ReunionTestInstanceRedirectedPhaseEventName";
+
+#ifndef WIDEN2
+#define WIDEN2(x) L ## x
+#endif
+
+#ifndef WIDEN
+#define WIDEN(x) WIDEN2(x)
+#endif
+
+#ifndef __WFILE__
+#define __WFILE__ WIDEN(__FILE__)
+#endif
+
+#ifndef __WFUNCTION__
+#if defined(_WIN32)
+#define __WFUNCTION__ WIDEN(__FUNCTION__)
+#else
+#define __WFUNCTION__ L""
+#endif
+#endif
+
+inline std::wstring StripNS(PCWSTR symbol)
+{
+    std::wstring result = symbol;
+    return result.substr(result.rfind(L":") + 1);
```

Comment:
>result.rfind(L":") + 1 [](http://example.com/codeflow?start=25&length=22)

If rfind doesn't find a match it returns `string::npos` == `size_t(~0)` == 0xFFFFFFF....

+1 turns that into 0

substr(0) keeps the whole string

Cryptic way to make "not found" an expensive NOP

CONSIDER: bit more straightforward logic